### DISCUSSION TITLE
Preserve CATALOG and ISRC tags from cuesheet in CHD

### DISCUSSION THREAD

**Discussion by tjanas** [mamedev/discussions#77] [General]
When creating a CHD file from a bin/cue that has CATALOG and ISRC values in the cuesheet, such as http://redump.org/disc/95434/, it does not seem to preserve this information, as it is not present in the cue generated by chdman extractcd. This information is important, and in specific cases like this, is required for the game disc image to function correctly.

**Comment by rb6502** [comment]
CHD was not intended to be an emulation-wide archival format and its current capabilities reflect that.  If someone wants to submit pull requests to improve this and other things people dislike about the current state I'll be happy to review them though.

**Comment by startaq** [comment]
I don't think we need special support for those, they will be stored as part of the subchannel.
